pages/api/assessments/index.js

```diff
@@ -0,0 +1,20 @@
+import Assessment from "../../../db/modules/Assessment.js";
+import dbConnect from "../../../db/connect.js";
+
+export default async function handler(request, response) {
+  // goal is to connect to the db
+  await dbConnect();
+
+  // check if the request method is GET
+
+  if (request.method === "GET") {
+    try {
+      const assessments = await Assessment.find();
+      console.log("assessments", assessments);
+      response.status(200).json(assessments);
+    } catch (error) {
+      console.error("Error fetching assessments:", error);
+      response.status(500).json({ error: "Internal Server Error" });
```

Comment:
Instead of a JSON response I get an HTML error page when the fetch is being performed.
Check your error message in the terminal where you started the app.

<img width="1168" alt="Screenshot 2023-12-15 at 17 17 18" src="https://github.com/JulianKle/capstone-project/assets/4458383/6af1bdc9-502f-456c-a7f9-9e34e233f8a6">